Kernel/Arch/aarch64/init.cpp

```diff
@@ -22,17 +22,17 @@
 #include <Kernel/Arch/aarch64/RPi/UART.h>
 #include <Kernel/Arch/aarch64/Registers.h>
 #include <Kernel/Arch/aarch64/TrapFrame.h>
-#include <Kernel/CommandLine.h>
+#include <Kernel/Boot/CommandLine.h>
 #include <Kernel/Devices/DeviceManagement.h>
 #include <Kernel/FileSystem/VirtualFileSystem.h>
 #include <Kernel/Graphics/Console/BootFramebufferConsole.h>
```

Comment:
(might belong more to newly created `Boot`?)

Reply:
I think it's OK as it is now, we can think about merging init code of both aarch64 and x86-64 later :)